File: Leetcode-Easy/9-PalindromeNumber/palindromenumber.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string xs=to_string(x);
        int i=0, j=xs.length()-1;
        while(i<=j)
        {
            if(xs[i]!=xs[j])
                return false;
            i++;
            j--;
        }
        return true;
    }
};

File: Leetcode-Easy/9-PalindromeNumber/palindromenumber.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        xs=str(x)
        i=0
        j=len(xs)-1
        while i<=j:
            if(xs[i]!=xs[j]):
                return False
            i+=1
            j-=1
        return True
